packages/core/test/combinator/chain-test.js

```diff
@@ -36,13 +36,13 @@ describe('chain', function () {
 
     return collectEvents(s, ticks(3))
       .then(events => {
-        assert.same(2, events.length)
+        is(2, events.length)
```

Comment:
I've started to prefer writing assertions like this:

```js
// Array of (time, value) pairs representing the events we expect
const expected = [{ time: 1, value: 1 }, { time: 2, value: 2 }]

// collectEventsFor is a convenience function on top of collectEvents()
return collectEventsFor(3, s).then(eq(expected))
```

Do you guys like that style also?